__tests__/app.test.js

```diff
@@ -44,3 +44,33 @@ describe("GET /api/", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  it("GET an article object with specific id", () => {
+    const articleId = 2;
```

Comment:
This is not a dynamic variable, there is no need to declare separately. a get request to /api/articles/2 is valid